README.md

```diff
@@ -2,20 +2,20 @@
 [![Molecule CI/CD](https://github.com/nginxinc/ansible-role-nginx-management-suite/workflows/Molecule%20CI/CD/badge.svg)](https://github.com/nginxinc/ansible-role-nginx-management-suite/actions/workflows/molecule.yml/badge.svg)
 [![License](https://img.shields.io/badge/License-Apache--2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-# Ansible NGINX Management Suite Role
+# Ansible NGINX Instance Manager (NIM) Role
```

Comment:
```suggestion
# Ansible NGINX Instance Manager Role
```

Reply:
FYI: Marketing says we can't abbreviate product names, unfortuantly. They need to be spelled out each time.